test/session.test.js

```diff
@@ -275,13 +275,15 @@ describe('server session', function() {
   });
 
   it('should fail if syncing while a session refresh is in progress', async function () {
-    this.application.apiService.refreshSession();
+    const refreshPromise = this.application.apiService.refreshSession();
     const syncResponse = await this.application.apiService.sync([]);
 
     expect(syncResponse.error).to.be.ok;
 
     const errorMessage = 'Your account session is being renewed with the server. Please try your request again.';
```

Comment:
Is there no string constant we can access from the test files? Having a test break because of a wording change doesn't sound ideal.

Reply:
We just capture the promise value so we can await it at the end of the test.

Reply:
I meant `const errorMessage`. But it looks like that belongs in a previous commit anyway.